modules/partials/header.php

```diff
@@ -1,57 +1,74 @@
-        <!-- The Header -->
-        <header> 
-        <div>
-            <h4>The logo</h4>
-        </div>
-        <div>
-                <h2 class="slogan">The header slogan</h2>
+  <head>
+    <meta charset="utf-8">
+    <title><?php echo $pageTitle; ?></title>
+    <link rel="stylesheet", href="./assets/css/index.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="./assets/js/index.js"></script>
+  </head>
+  <body>
+    <header>
+        <div id="logo">
+            <h4>The logo<?php echo $userId; ?></h4>
         </div>
+        <div id="slogan">The header slogan</div>
         <div id="form">
             <ul>
-                <li>Hi <span>Guest</span></li>
-                <li><a href="javascript:void(0)" onclick="showLoginForm()">Login</a></li>
+                <li>Hi <span><?php echo $fullname; ?></span></li>
+                <li><a href=<?php if ($loginOrLogout === 'Login') {echo "javascript:void(0)";} else {echo "./index.php?m=logout";} ?> id="form1" onclick=<?php if ($loginOrLogout === 'Login'){echo "showLoginForm()";} ?>><?php echo $loginOrLogout ?></a></li>
             </ul>
-            
-            <form id="login">
-                <input type="text" name="username" placeholder="User name" />
-                <input type="password" name="password" placeholder="Password"/>
-                <label><input type="checkbox" name="rememberUsername" />Remember user name </label>
-                <button type="submit" name="Login">Login</button>
+            <form id="login" action="./index.php?m=login" method="post">
+                <input type="text" name="username" placeholder="User name">
+                <input type="password" id="login1" name="password" placeholder="Password">
+                <i class="material-icons" id="passwordIcon" onclick="hidePassword()">remove_red_eye</i>
+                <label><input type="checkbox" name="rememberUsername">Remember user name</label>
+                <button type="submit" name="login">Login</button>
             </form>
             <form method="GET" id="search">
                 <input type="text" name="keyword" />
                 <i class="material-icons">search</i>
             </form>
         </div>
-        </header>
-        
-        <!-- The menu -->
-        <nav>
-            <ul>
-                <li><a href="./index.php">Home</a></li>
-                <li><a href="./index.php?m=profile">My Profile</a></li>
-            </ul>
-        </nav>
+    </header>
+    <nav>
+        <ul>
+            <li><a href="./index.php">Home</a></li>
+            <li><a href="./index.php?m=register"><?php if ($loginOrLogout === 'Login') {echo "Register";} ?></a></li>
+            <li><a href="./index.php?m=profile"><?php if ($loginOrLogout === 'Logout') {echo "My Profile";} ?></a></li>
+            <li><a href="./index.php?m=change_password"><?php if ($loginOrLogout === 'Logout') {echo "Change Password";} ?></a></li>
```

Comment:
Move all `if...else...` to outside of `li`, you shouldn't output empty `li`. And try to make the source code more simple.